op-supervisor/metrics/metrics.go

```diff
@@ -14,8 +15,10 @@ type Metricer interface {
 	RecordUp()
 
 	opmetrics.RPCMetricer
-	RecordCrossUnsafeRef(chainID eth.ChainID, r eth.BlockRef)
-	RecordCrossSafeRef(chainID eth.ChainID, r eth.BlockRef)
+	RecordCrossUnsafeRef(chainID eth.ChainID, r types.BlockSeal)
+	RecordCrossSafeRef(chainID eth.ChainID, r types.BlockSeal)
+	RecordLocalSafeRef(chainID eth.ChainID, r types.BlockSeal)
+	RecordLocalUnsafeRef(chainID eth.ChainID, r types.BlockSeal)
```

Comment:
let's take the opportunity to order them local-unsafe, cross-unsafe, local-safe, cross-safe, and remove the superfluous `Ref` suffix
```suggestion
	RecordLocalUnsafe(chainID eth.ChainID, r types.BlockSeal)
	RecordLocalSafe(chainID eth.ChainID, r types.BlockSeal)
	RecordCrossUnsafe(chainID eth.ChainID, r types.BlockSeal)
	RecordCrossSafe(chainID eth.ChainID, r types.BlockSeal)
```